src/plugins/intel_gpu/src/graph/graph_optimizer/set_required_layouts.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include "pass_manager.h"
+#include "data_inst.h"
+#include "mutable_data_inst.h"
+#include "program_node.h"
+#include "intel_gpu/runtime/engine.hpp"
+#include "runtime/cldnn_itt.hpp"
+#include <iostream>
+#include "to_string_utils.h"
+#include "intel_gpu/runtime/debug_configuration.hpp"
+#ifdef ENABLE_ONEDNN_FOR_GPU
+#include <oneapi/dnnl/dnnl.hpp>
+#include "impls/onednn/utils.hpp"
+#include "impls/onednn/convolution_onednn.hpp"
+#endif
+
+using namespace cldnn;
+
+void set_required_layouts::run(program& p) {
+    OV_ITT_SCOPED_TASK(itt::domains::CLDNN, "CLDNN::pass::SetRequiredLayouts");
```

Comment:
`itt::domains::intel_gpu_plugin, "pass::set_required_layouts"`

Reply:
OK I will update it.